CompileIO/compile.io
# 88 professor class creationv2

# Addressed Issue(s)

#88 

# Goals

Have a professor create a course

# Special Considerations

Mongo Needs to be running, you should probably delete your database by doing 
db.Assignment.drop()
db.Student.drop()
db.Section.drop()
db.Code.drop()
db.Professor.drop()
db.Course.drop() 

# Acceptance Criteria

If you can create and update a section, and add or delete students to the section

# Testing

- Create a class
- Create a section in that class
- Add Students to the section
- submit the section
- Go to the section you just created, and add or remove students 
- click submit, and then see if your changes worked


# Branch

88-professor-class-creationv2
